server/src/main/java/org/candlepin/controller/Refresher.java

```diff
@@ -40,6 +43,7 @@
 
     private CandlepinPoolManager poolManager;
     private SubscriptionServiceAdapter subAdapter;
+    @Inject private OwnerManager ownerManager;
```

Comment:
Is the constructor for this class not injected?  I'm a little wary of field injection since it relies on reflection tricks and stuff (and is ultimately less transparent than constructor injection which is how most of the rest of our stuff is).